pkg/builder/build.go

```diff
@@ -51,35 +44,59 @@ type Builder struct {
 	mgr            manager.Manager
 	predicates     []predicate.Predicate
 	managedObjects []runtime.Object
+	watchRequest   []watchRequest
 	config         *rest.Config
 	ctrl           controller.Controller
 }
 
-// SimpleController returns a new Builder
+// SimpleController returns a new Builder.  Deprecated - call Controller(Manager) instead.
 func SimpleController() *Builder {
 	return &Builder{}
 }
 
-// ForType sets the ForType that generates other types
+// Controller returns a new controller builder that will be started by the provided Manager
+func Controller(m manager.Manager) *Builder {
+	return SimpleController().WithManager(m)
+}
+
+// ForType defines the type of Object being *reconciled*, and configures the Controller to respond to create / delete /
+// update events by *reconciling the object*.
+// This is the equivalent of calling
+// Watch(&source.Kind{Type: apiType}, &handler.EnqueueRequestForObject{})
 func (b *Builder) ForType(apiType runtime.Object) *Builder {
 	b.apiType = apiType
 	return b
 }
 
-// Owns configures the Application Controller to respond to create / delete / update events for objects it managedObjects
-// - e.g. creates.  apiType is an empty instance of an object matching the managed object type.
+// Owns defines types of Objects being *generated* by the Controller, and configures the Controller to respond to
+// create / delete / update events by *reconciling the owner object*.  This is the equivalent of calling
+// Watch(&handler.EnqueueRequestForOwner{&source.Kind{Type: <ForType-apiType>}, &handler.EnqueueRequestForOwner{OwnerType: apiType, IsController: true})
 func (b *Builder) Owns(apiType runtime.Object) *Builder {
 	b.managedObjects = append(b.managedObjects, apiType)
 	return b
 }
 
+type watchRequest struct {
+	src          source.Source
+	eventhandler handler.EventHandler
+}
+
+// Watch exposes the lower-level Controller Watch functions through the builder.  Consider using
+// Owns or ForType instead of Watch directly.
+func (b *Builder) Watch(src source.Source, eventhandler handler.EventHandler) *Builder {
```

Comment:
nit: follow the same phrasing as `ForType` or `Owns` here -- `Watches` or something similar

Reply:
Done